usr/share/rear/output/default/950_copy_result_files.sh

```diff
@@ -99,8 +99,8 @@ case "$scheme" in
         # FIXME: Verify if usage of $array[*] instead of "${array[@]}" is actually intended here
         # see https://github.com/rear/rear/issues/1068
         LogPrint "Copying result files '${RESULT_FILES[*]}' to $scheme location"
-        Log "lftp -c open $OUTPUT_URL; mput ${RESULT_FILES[*]}"
-        lftp -c "open $OUTPUT_URL; mput ${RESULT_FILES[*]}" || Error "Problem transferring result files to $OUTPUT_URL"
+        Log "lftp -c open $OUTPUT_URL; $LFTP_PARAMS mput ${RESULT_FILES[*]}"
+        lftp -c "open $OUTPUT_URL; $LFTP_PARAMS mput ${RESULT_FILES[*]}" || Error "Problem transferring result files to $OUTPUT_URL"
```

Comment:
@vigri 
I would appreciate it if you also could improve our old and rather meaningless
Error message into something that is more helpful for the user, e.g. something like
```
lftp -c "open $OUTPUT_URL; $LFTP_PARAMS mput ${RESULT_FILES[*]}" || Error "lftp failed to transfer '${RESULT_FILES[*]}' to '$OUTPUT_URL' (lftp exit code is $?)"
```
or whatever you like that could help the user to understand what went wrong.